plugins/jobs/src/js/JobsRunNow.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/do";
+import "rxjs/observable/empty";
```

Comment:
I would only leave here the rxjs imports we use (which probably means removing all the operator/observable imports).